dbld/images/centos7/dev-dependencies.txt

```diff
@@ -0,0 +1,54 @@
+# required for autogen
+autoconf
+autoconf-archive
+automake
+libtool
+pkgconfig
+# required for configure
+bison
+flex
+libcap-devel
+libdbi-devel
+libesmtp-devel
+glib2-devel
+libnet-devel
+python-devel
+openssl-devel
+systemd-devel
+tcp_wrappers-devel
+libcurl-devel
+#ivykis-devel
+json-c-devel
+ivykis-devel
+gradle
+libdbi
+libdbi-drivers
+librdkafka
+riemann-c-client-devel
+syslog-ng-java-deps
+eventlog-devel
+autoconf-archive
+GeoIP-devel
+python-pep8
+libesmtp-devel
+mongo-c-driver-devel
+hiredis-devel
+python-pep8
+cyrus-sasl-devel
+rpm-build
+java-1.8.0-openjdk-devel
+libuuid-devel
+# required for make
+make
+# required for make check
+#criterion-devel
+# other tools
+gdb
+git
+lsof
+strace
+vim
+# debian packaging tools
+docbook-xsl
+python-nose
+python-ply
```

Comment:
how did you get the rpm packaging stuff? this seems to be the "old" one on the master branch.

you'd need to merge in the release branch, as both debian/rpm packaging is on those branches. It's a hassle, which we should discuss with @mochrul and @lbudai.

while staying on the "docker-support" branch, just do "git merge release" and you'd have all the files. Later you can reset them away.